packages/angular/pwa/pwa/files/assets/manifest.json

```diff
@@ -3,7 +3,7 @@
   "short_name": "<%= title %>",
   "theme_color": "#1976d2",
   "background_color": "#fafafa",
-  "display": "browser",
+  "display": "fullscreen",
```

Comment:
Please leave the display as browser for the default behavior. Changing it to `fullscreen` could be added as an option for this schematic...
`ng add @angular/pwa --dispaly fullscreen`
But the expected behavior for most users right now should remain as browser.